data/tools/wmlindent

```diff
@@ -103,6 +103,29 @@ class bailout(Exception):
         self.lineno = lineno
         self.msg = msg
 
+def strip_comment(line, in_string):
+    """
+    Strip a trailing comment from a line, if present.
+    Allows for unbalanced quotes, where comment characters are literal.
+    """
+    if not line:
+        return line
+    split = line.split('"')
```

Comment:
Does this properly handle doubled double quotes in strings?
Like: `attr="foo ""bar"" baz"`

Reply:
I believe so, but I'm not sure if there were any of those in my testing cases…

Reply:
Just checked. Python's string.split() does indeed yield empty elements, so for example:

`some_key="This is a ""test""!!!" # With a comment at the end`

would split to 

`['some_key=', 'This is a', '', 'test', '', '!!!', ' # With a comment at the end']`

And from there the logic should work correctly.